dockerfiles/init/manifests/che.env

```diff
@@ -101,7 +101,16 @@
 #     agents within the workspace when it boots, then the workspace will not be started.
 #CHE_MACHINE_WS_AGENT_MAX_START_TIME_MS=300000
 
-
+# Workspace Agent Java Options
+#   Java command line options will be exported as a value of the JAVA_OPTS
+#   environment variable in the 'development' machine.
+#CHE_WORKSPACE_JAVA__OPTS=-Xms256m -Xmx2048m -Djava.security.egd=file:/dev/./urandom
```

Comment:
We have a problem now in that the Java and Maven options have different formats.
CHE_WORKSPACE_MAVEN_OPTIONS
CHE_WORKSPACE_JAVA__OPTS

Admins will expect common syntax, so propose:
CHE_WORKSPACE_MAVEN_OPTIONS
CHE_WORKSPACE_JAVA_OPTIONS